teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -158,3 +158,933 @@ With regards to this checklist, reviewers should do the following:
   * Activities: Predetermined amounts
 * [ ] Have curriculum proofread by writing team \(currently Victoria Xu @vkxu657\)
 
+### Stage 3 Documentation \(Atul\)
+
+> Note: All elements of a unit \(Module,Activity,Topic,etc.\) are REQUIRED unless stated otherwise.
+
+#### Topic README Format
+
+**Github ID**
```

Comment:
a note should be added here: there is a maintained Airtable for Github IDs. Devs should contact their managers and if the github ID is missing, then Kevin should be contacted.